examples/ex1.cpp

```diff
@@ -112,6 +112,7 @@ int main(int argc, char *argv[])
       }
    }
 
+#if 0
```

Comment:
Can you please revert these and re-merge in `next`?

Reply:
Done, but I can't push to `next`:
```
remote: error: You're not authorized to push to this branch.
```

Reply:
Ok, I pushed. Can you make sure you are in the LLNL org + you have 2FA enabled? That's required to be able to push to `next` now because of GitLab.